noetic/distribution.yaml

```diff
@@ -944,6 +944,26 @@ repositories:
       url: https://github.com/ros-gbp/libg2o-release.git
       version: 2020.5.3-1
     status: maintained
+  marker_msgs:
+    doc:
+      type: git
+      url: https://github.com/tuw-robotics/marker_msgs.git
+      version: noetic-devel
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/tuw-robotics/marker_msgs-release.git
+      version: 0.0.6-0
+  marker_rviz_plugin:
+    doc:
+      type: git
+      url: https://github.com/tuw-robotics/marker_rviz_plugin.git
+      version: noetic-devel
+    source:
+      type: git
+      url: https://github.com/tuw-robotics/marker_rviz_plugin.git
+      version: noetic-devel
+    status: maintained
```

Comment:
Did you also want a `release` section for `marker_rviz_plugin`? I ask because I see a Noetic release in the release repo: https://github.com/tuw-robotics/marker_rviz_plugin-release#marker_rviz_plugin-noetic---002-1

Reply:
Yes, is there a problem

Reply:
> Yes, is there a problem

Two minor ones. First is a merge commit, but that's easy to fix. Second is this PR is only adding `source` and `doc` entries for `marker_rviz_plugin`, which is fine if that's what was intended, but it seems like the intent was to add a `release` section for it to.

Reply:
@maxbader friendly ping, mind fixing the merge conflict? Also, if you want to release `marker_rviz_plugin` then the PR will need a `release` block for that repository. If you only want to add it to the doc/source index then the PR is fine once the merge conflict is fixed.

Reply:
I will do it on the end of this week. Sorry for the delay.

Reply:
@maxbader friendly ping :)

Reply:
Sorry for not finding the time to finish the release. 
I think it is better to release first the marker_msgs and in some time the rviz plugin for visualization.